Assets/QA/Input_Test/Editor/ImportInputManagerAsset.cs

```diff
@@ -0,0 +1,53 @@
+﻿using System.IO;
+using UnityEditor;
+using UnityEngine;
+
+[InitializeOnLoad]
+public class ImportInputManagerAsset {
```

Comment:
Please run `format.sh` or `format.ps1` before committing. Otherwise the next time someone does, your files will get picked up, too.